release_notes_generator/action_inputs.py

```diff
@@ -159,14 +164,22 @@ def get_row_format_issue() -> str:
         """
         Get the issue row format for the release notes.
         """
-        return get_action_input(ROW_FORMAT_ISSUE, "{number} _{title}_ in {pull-requests}").strip()
+        if ActionInputs._row_format_issue is None:
```

Comment:
This is little confusing, I would add more of doc into method docstring probably. Because by simple reading you are saying "if __row_format_issue is None THAN I call func __clean_row_format_invalid_keywords". It is None, so you can't clean by your naming anything. Same in the method below for PR.

Reply:
This is quite simple. The internal value is None after initialization. The method checks if the value is None (if True, we do the first delivery), then calls get_action_input with the defined default value. The returned value is then cleaned for unknown inputs.
The get method doc string correctly describes what is delivered. The programmer can read what is done.